docs/docs-content/clusters/public-cloud/azure/azure-disk-encryption.md

```diff
@@ -0,0 +1,353 @@
+---
+sidebar_label: "Azure Disk Encryption"
+title: "Azure Disk Encryption"
+description:
+  "How to enable disk encryption on your Azure IaaS cluster using Disk Encryption Sets with customer-managed keys"
+icon: ""
+hide_table_of_contents: false
+tags: ["public cloud", "azure", "encryption", "security"]
+sidebar_position: 12
+---
+
+Palette supports disk encryption of your Azure Kubernetes cluster using Disk Encryption Sets with
+[customer-managed keys](https://learn.microsoft.com/en-us/azure/virtual-machines/disk-encryption#customer-managed-keys).
+
+By default, Azure encrypts all managed disks with
+[platform-managed keys](https://learn.microsoft.com/en-us/azure/virtual-machines/disk-encryption#platform-managed-keys),
+however, customer-managed keys enable you to have greater control over your key management.
+
+You can use Disk Encryption Sets to encrypt your nodes' Operating System (OS) and data disks by including a preset
+option in the <VersionedLink text="Palette eXtended Kubernetes (PXK)" url="/integrations/packs/?pack=kubernetes" />
+pack.
+
+## Limitations
+
+- Azure Disk Encryption is available from [4.5.a](../../../release-notes/release-notes.md) onwards.
```

Comment:
You don't need to do this as our docs are versioned. This content will only be available in new versions of our docs. 

```suggestion
```